include/gc/Dialect/Microkernel/MicrokernelEnum.td

```diff
@@ -0,0 +1,26 @@
+//===- MicrokernelEnum.td - microkernel dialect enum -------*- tablegen -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MICROKERNEL_ENUM
+#define MICROKERNEL_ENUM
+
+include "mlir/IR/EnumAttr.td"
+include "gc/Dialect/Microkernel/MicrokernelDialect.td"
+
+def Microkernel_BrgemmFlags : I64EnumAttr<
+    "BrgemmFlags", "see: microkernel_brgemm_flags",
```

Comment:
This either needs a better link or moving the description here. IMO the latter is preferrable.

Reply:
Description added here.